internal-resources/git-workflow-and-contributor-guide.adoc

```diff
@@ -4,135 +4,226 @@
 :numbered:
 :internal-guide:
 
-= Stonesoup contributor guide
+= {Product_Name} Contributor Guide
 
-== Overview
+== How to use this guide
 
-This document describes the tasks needed to contribute to stonesoup documentation.
+The Contributor Guide is for writers on the {Product_Name} team, or anyone who needs to edit our documentation.
 
-IMPORTANT: If at *ANY TIME* you feel lost or don't know what to do, start asking questions on the _#forum-hac-docs_ slack channel.
+NOTE: To get help at any point, post in the _#forum-hac-docs_ Slack channel.
 
-* Before you begin, you need to perform the following one time process to xref:install_and_configure_git[install and configure Git]
+We organized this guide into the following sections:
 
-* Once you have configured *Git* and set up your repositories, use this xref:documentation_update_process[documentation update process] to make changes to the stonesoup documentation.
+* *Getting started*: the Getting started section outlines all of the tools that you need to install before you can begin working on our documentation. The steps in this section should be a one-time process.
```

Comment:
In the second instance of getting started, why is the G in Getting capitalized but not the S in started? 

Shouldn't it be:

`*Getting started*: the getting started section outlines all of the tools`

Reply:
I was referencing it as a title, so wouldn't that be capital? Sort of like a proper noun